README.md

```diff
@@ -1,2 +1,16 @@
-# 2017-Hestia-Server
+push# 2017-Hestia-Server
 Home automation server.
+
+Tested on python version 3.5.2
+
+Before use first install `pip`
+`pip install flask`
+`pip install flask_restplus`
+
+Run the webservice using `python3 source/application.py`
+
```

Comment:
We already created an issue for this as it is a known problem, see #14. I will add this link to it as well. It will be handled in the next iteration.